recipes/systemd/all/conanfile.py

```diff
@@ -0,0 +1,57 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+
+
+class SystemdConan(ConanFile):
+    name = "systemd"
+    version = "system"
+    description = "System and service manager"
+    topics = ("systemd", "udev")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://systemd.io/"
+    license = "LGPL-2.1-or-later"
+    settings = {"os": "Linux"}
+
+    def system_requirements(self):
+        if tools.os_info.is_linux and self.settings.os == "Linux":
+            package_tool = tools.SystemPackageTool(conanfile=self, default_mode="verify")
+            if tools.os_info.with_yum or tools.os_info.with_dnf:
+                packages = ["systemd", "systemd-libs", "libudev-devel"]
+            elif tools.os_info.with_apt:
+                packages = ["libsystemd-dev", "libudev-dev", "systemd"]
```

Comment:
```suggestion
                packages = ["libudev-dev", "systemd"]
                if not (tools.os_info.linux_distro == "ubuntu" and tools.os_info.os_version < "15"):
                    packages.append("libsystemd-dev")
```
fix ubuntu trusty, which does not have libsystemd-dev